packages/utils/src/libp2p/addressSorters.ts

```diff
@@ -2,14 +2,18 @@ import { Address } from 'libp2p/src/peer-store'
 import isIpPrivate from 'private-ip'
 import { Multiaddr } from 'multiaddr'
 
-export function isMultiaddrPrivate(multiaddr: Multiaddr): boolean {
-  const { address } = multiaddr.nodeAddress()
-  return isIpPrivate(address)
+export function isMultiaddrLocal(multiaddr: Multiaddr): boolean {
+  try {
+    const { address } = multiaddr.nodeAddress()
```

Comment:
Instead of waiting for an exception, you could also use `multiaddr.tuples` and switch according to first protocol